src/BigQuery/Job.php

```diff
@@ -121,7 +127,8 @@ public function cancel(array $options = [])
     }
```

Comment:
Should the example for the `cancel` method be updated to use the new `wait` function?